src/WinRT.Runtime/Interop/IID.tt

```diff
@@ -39,18 +39,25 @@ var entries = new (string Name, string IID, bool IsPublic)[]
     ("ISupportErrorInfo", "DF0B3D60-548F-101B-8E65-08002B2BD119", false),
     ("ILanguageExceptionErrorInfo", "04A2DBF3-DF83-116C-0946-0812ABF6E07D", false),
     ("IRestrictedErrorInfo", "82BA7092-4C88-427D-A7BC-16DD93FEB67E", false),
-    ("INotifyCollectionChangedEventArgsFactory", "5108EBA4-4892-5A20-8374-A96815E0FD27", false),
-    ("INotifyCollectionChangedEventArgs", "DA049FF2-D2E0-5FE8-8C7B-F87F26060B6F", false),
-    ("PropertyChangedEventArgsRuntimeClassFactory", "7C0C27A8-0B41-5070-B160-FC9AE960A36C", false),
+    ("MUX_INotifyPropertyChanged", "90B17601-B065-586E-83D9-9ADC3A695284", false),
+    ("WUX_INotifyPropertyChanged", "CF75D69C-F2F4-486B-B302-BB4C09BAEBFA", false),
+    ("MUX_INotifyCollectionChanged", "530155E1-28A5-5693-87CE-30724D95A06D", false),
+    ("WUX_INotifyCollectionChanged", "28B167D5-1A31-465B-9B25-D5C3AE686C40", false),
+    ("MUX_INotifyCollectionChangedEventArgsFactory", "5108EBA4-4892-5A20-8374-A96815E0FD27", false),
+    ("WUX_INotifyCollectionChangedEventArgsFactory", "B30C3E3A-DF8D-44A5-9A38-7AC0D08CE63D", false),
+    ("MUX_INotifyCollectionChangedEventArgs", "DA049FF2-D2E0-5FE8-8C7B-F87F26060B6F", false),
+    ("WUX_INotifyCollectionChangedEventArgs", "4CF68D33-E3F2-4964-B85E-945B4F7E2F21", false),
+    ("MUX_NotifyCollectionChangedEventHandler", "CA10B37C-F382-4591-8557-5E24965279B0", false),
+    ("WUX_NotifyCollectionChangedEventHandler", "8B0909DC-2005-5D93-BF8A-725F017BAA8D", false),

```

Comment:
Is this the WUX one for the MUX IID?  Just noticed it matched up with the previous iid for this.

Reply:
Yep these need to be swapped. I checked the rest and they should be fine.

Reply:
Thank you for the quick fix Jeremy! 😄